### ISSUE TITLE
Upgrade to version 4 of the API

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#4485]
We use the API for a few things. Let's analyze what they are and then get them migrated to version 4.

Here's the migration guide: 

https://www.courtlistener.com/help/api/rest/v4/migration-guide/

It should have all the information we need to do this upgrade. If it doesn't, upgrading it should be part of this ticket too.

@quevon24, I'm thinking in particular of your data import script, so maybe this is a good issue for you to take on, but step 1 is to figure out all the places we use v3.

**Comment by mlissner** [comment]
@flooie do you for triage, please. :)

**Comment by quevon24** [comment]
Yes, we need to update the command to clone items from the API, I already had it pending in my sticky notes, i'll create an issue for that. 

I'll give the migration guide a careful read to get an idea of ​​how many changes need to be made, maybe it's even an opportunity in case someone has requests for information that can be cloned and that would be useful for developers.

**Comment by mlissner** [comment]
I don't think you need a fresh issue just for the clone command; we can do that here, but I would like to also use this issue to find ALL other uses of v3 so we can discuss them. 

Thank you!

**Comment by quevon24** [comment]
I saw that there are several tests for v3 but not for v4, and I don't think it's used anywhere else except in the command I mentioned before. I'll check again to see if I find anything else.

**Comment by mlissner** [comment]
Seems simple enough. Will be good to get this upgraded.
